lte/gateway/c/oai/protos/mme_nas_state.proto

```diff
@@ -24,20 +24,19 @@ message BearerContext {
   uint32 qci = 5; // qci_t
 
   uint32 pdn_cx_id = 6; // pdn_cid_t
-  uint32 bearer_state = 7; // mme_app_bearer_state_t
-  EsmEbrContext esm_ebr_context = 8;
-  Fteid enb_fteid_s1u = 9;
-
-  uint32 priority_level = 10; // priority_level_t
-  uint32 preemption_vulnerability = 11;
-  uint32 preemption_capability = 12;
-  gateway.spgw.TrafficFlowTemplate saved_tft = 13;
-  BearerQos saved_qos = 14;
+  EsmEbrContext esm_ebr_context = 7;
+  Fteid enb_fteid_s1u = 8;
+
+  uint32 priority_level = 9; // priority_level_t
+  uint32 preemption_vulnerability = 10;
+  uint32 preemption_capability = 11;
+  gateway.spgw.TrafficFlowTemplate saved_tft = 12;
+  BearerQos saved_qos = 13;
```

Comment:
It is not a good practice to change the protobuf field numbers. Simply deleting the no longer needed field should suffice. Any new addition can be assigned the next count.